internal/dnfjson/cache.go

```diff
@@ -1,46 +1,194 @@
 package dnfjson
 
 import (
+	"fmt"
 	"io/fs"
 	"os"
 	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/gobwas/glob"
 )
 
+// A collection of directory paths, their total size, and their most recent
+// modification time.
+type pathInfo struct {
+	paths []string
```

Comment:
For our purposes, could would it make sense to drop this and instead do glob `<repoid>*` at runtime? Not sure it makes a big difference, just always think computing things is better than storing them.

Reply:
Yeah that's a nice idea.  Less bookkeeping.